### DISCUSSION TITLE
Welcome to project-layout Discussions!

### DISCUSSION THREAD

**Discussion by kcq** [golang-standards/project-layout#242] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Abedmuh** [comment]
Can we have simple example running project with full implementation of this template structure...?
